cmake/BLTOptions.cmake

```diff
@@ -50,6 +50,12 @@ option(ENABLE_BENCHMARKS "Enables benchmarks" OFF)
 option(ENABLE_COVERAGE   "Enables code coverage support" OFF)
 
 
+################################
+# TPL Options
+################################
+option(ENABLE_GIT       "Enables Git support" ON)
```

Comment:
Could we expand the string a bit? Perhaps "Enables git-related macros in blt".

Reply:
It's not just the macros in BLT that it is turning off/on.  It also is enables git in the codes build system as well via Git_FOUND/GIT_EXECUTABLE.